### DISCUSSION TITLE
[Arena][PVP] Add daily quest that rewards arena points.

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#7045] [Suggestions]
I've seen this implemented on another server in the following way:

- **Quest:** Play 10 matches and secure 1 win to complete the quest.
- **Rewards:** 25 arena points.

This approach addresses three key issues:

1. **Increased Arena Participation:** Encourages more players to participate.
2. **Diverse Ladder:** More teams queuing leads to a more diverse ladder.
3. **Gear Progression:** With T6 on the horizon, the current gear progression feels slow. As of the date of writing, even high-rated (rank1/rank2) players have only one item. If we compare to PvE gear progression, It's often much faster to gear your character.

**Comment by Never6** [comment]
sure

**Comment by Ottawia** [comment]
In my opinion, a weekly quest would be better in this case, especially since not everyone has time to play 10 arenas daily, and the server is small (of course, more points for completion and maybe more arenas to play so that would be more than just a weekly cap)

**Reply by minthes** [reply]
Tbh, the primary goal should be to always increase participation over catering to people with limited schedules. If we want people who can't play a lot to also get gear, we would implement the WoTLK classic system where you get 1k points per week for just playing cap. I would also support this as well, as someone who plays hybrids in pvp. It would actually be realistic for me to gear 2 specs. But that's neither here nor there. 

The point is, the daily quest is primarily concerned with increasing participation and to diversify the ladder. Which doesn't align with catering to people who have limited time to play.

**Reply by tbcxd** [reply]
I'm all for daily and not weekly.
A weekly wouldnt change much of the activity.
So what someone cant get their games today? It's all good, they will get them tomorrow or the day after.

**Reply by Franaloudzx** [reply]
If it's gonna be weekly, this quest isn't gonna change the main problem: that ppl will only queue for their 10 weekly games.

Also if it's weekly i would agree with 25 or even more arena points.

i would also require 10 wins at least for weekly quest, so ppl have to queue more than the average "10 games per week", plus the fact meritocracy exists when someone is queueing for rewards and not to gift free rating.

I would also make the same quest but for double reward, only obtainable doing 3vs3.

**Comment by Bijjoo** [comment]
I believe that it is necessary to introduce 3x3 solo que on Chromiecraft, this will attract many pvp players to the game and create an active pvp environment for the pvp community. I also suggest doing a daily quest for 5 victories in a solo que, which will give 15-25 arena points as a reward.

**Reply by tbcxd** [reply]
3v3 soloq would mean getting rid of one of the brackets, you gonna get some gladiators mad if you do that:^)

Srsly now, I don't think soloq would be beneficial, seems hard to script and we gonna end up with the same issue as bgs with class stacking etc.
3 prot warrs 3s would be awful, lol.

**Reply by Never6** [reply]
Gus Fring is impressed, he applauds your bravery

**Reply by Bijjoo** [reply]
No need to invent something new, just look at how it works on Warmane or Endless - just perfect, you need to do the same in Chromiecraft. On these servers, 3s solo q is very popular and players play non stop.

**Reply by heyitsbench** [reply]
> No need to invent something new [...]

You do realize Warmane/Endless code is not public, and just looking (from a player's perspective) is not enough to recreate their system, right? Not to mention the obvious difference between population that inherently aids a solo queue system.

**Reply by Bijjoo** [reply]
Chromiecraft introduced a triple hardcore system to cater to the PvE public, so maybe it's worth making an effort to attract more PvP oriented players as well? Considering that the working model already exists and there is no need to rack your brains over which concept to choose. You're probably a pve player and you just don't understand how important this option is for pvp players to be able to play 3s solo que on the Chromiecraft server. This will significantly boost the activity of the pvp community on the server. If you're just being complaining about population and doing nothing to attract more players, then you won't get nothing.

**Reply by heyitsbench** [reply]
> Considering that the working model already exists and there is no need to rack your brains over which concept to choose.

And as tbcxd mentioned, the current model (I assume you mean for BGs) tends to have issues with class stacking, e.g. throwing all the rogues on one team.

> You're probably a pve player and you just don't understand how important this option is [...]

Speaking as a PvE player, I don't hate the idea of solo queue. Depending on how it would be implemented and how it might affect my existing arena team play, I'd be quite inclined to use it. Don't assume just because someone prefers one side of the game that they have no clue how other aspects work.

I mentioned the population issue because there *is* a population issue, and there is a lot that makes that a very difficult challenge to overcome.

**Reply by Never6** [reply]
common bench W

**Comment by Never6** [comment]
> Chromiecraft introduced a triple hardcore system to cater to the PvE public, so maybe it's worth making an effort to attract more PvP oriented players as well? Considering that the working model already exists and there is no need to rack your brains over which concept to choose. You're probably a pve player and you just don't understand how important this option is for pvp players to be able to play 3s solo que on the Chromiecraft server. This will significantly boost the activity of the pvp community on the server. If you're just being complaining about population and doing nothing to attract more players, then you won't get nothing.

There is no reason to be so condescending  to other players
Soloq would boost activity but i highly doubt it boosts population, "Competitive" twinking was always a niche thing and not everyone was into that sort of thing. So it would boost arena activity but its not gonna boost population when a vast majority of the players quite frankly dont give a shit about pvp.
Even if they could implement soloq (not to be rude but I doubt they can or even have the drive to do so) the matchmaking its so broken and stacks classes that it wouldnt even be enjoyable
A weekly/daily quest for 10 games is the only correct choice in this. As anything else is just bound to frustrate more and more people into just not pvping anymore (which most are)

**Reply by Franaloudzx** [reply]
I support this idea 100%. The Solo Queue system is what brings activity to many games, the fact that u don't need to spend time looking for ppl to queue and is all automatized is what made it so successful. So yeah...is a must from PvP perspective. Also is really fun to be paired and adapt ur gameplay to different setups.

Would be top addition.

I would pair it by Melee / Caster / Healer.

**Comment by TotallyNotMyAccount** [comment]
Very much in favor of this. Also, 3v3 soloQ is a completely different topic, and should get it's own thread.
